packages/lib/src/utils/Script.ts

```diff
@@ -33,7 +33,10 @@ class Script {
 
             this.script.src = this.src;
             this.script.async = true;
-            this.script.onload = resolve;
+            this.script.onload = event => {
+                this.script.setAttribute('data-script-loaded', 'true');
```

Comment:
Maybe we could rename this attribute to something more component-specific like `data-paypal-script-loaded`. This way we could use a similar approach in other components if needed.

Reply:
I'd rather keep it generic as it is now. The `Script` class is a generic class, and it is used by other UI components. We shouldn't couple it to PayPal here.

This code fixes that issue which the `resolve` wasn't being called when loading the script twice. Reason for that is that the script was already loaded, so the event listener isn't fired again. 

https://github.com/Adyen/adyen-web/blob/341b082213b480a36692c9ec7fe22468834223a5/packages/lib/src/utils/Script.ts#L51

Proposed solution here is that as soon as the script is loaded, we flag is as loaded, and then if it gets called again, we simply resolve the promise instead of adding an event listener which will never be called.

I believe that other payment methods will benefit from this change. Btw, I tested with GooglePay locally and it worked fine.

Reply:
Ah, my mistake I see the change is done in Script.ts directly. Thanks for the clarification.